mayastor/src/replica.rs

```diff
@@ -369,6 +394,37 @@ impl Replica {
             .send(errno_result_from_i32(lvol_ptr, errno))
             .expect("Receiver is gone");
     }
+
+    /// Callback called from SPDK for snapshot create method.
+    extern "C" fn snapshot_done_cb(
```

Comment:
Please consider to define the callback in the same function you use it. i.e fn something() { extern "C " fn(); // rest of code } this keep it nice and clean

Reply:
OK, this looks like a more recent style, the other callbacks in the same file don't do that :)